code/game/objects/items/melee/misc.dm

```diff
@@ -1079,3 +903,92 @@
 			target.LAssailant = user
 
 		cooldown = world.time + 15
+
+//Former Wooden Baton
+/obj/item/melee/tonfa
+	name = "Police Tonfa"
+	desc = "A traditional police baton for gaining the submission of an uncooperative target without the use of lethal-force. \
+		As with all traditional weapons, the target will find themselves bruised, but alive. It has proven to be effective in preventing \
+		repeat offenses and has brought employment to lawyers for decades."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "beater"
+	item_state = "beater"
+	worn_icon_state = "classic_baton"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	force = 12
+	throwforce = 7
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_LARGE
+	hitsound = 'sound/effects/woodhit.ogg'
+	/// Damage dealt while on help intent
+	var/non_harm_force = 4
+	/// Stamina damage dealt
+	var/stamina_force = 25
+
+/obj/item/melee/tonfa/attack(mob/living/target, mob/living/user)
+	var/target_zone = user.get_combat_bodyzone(target)
+	var/armour_level = target.getarmor(target_zone, STAMINA, penetration = armour_penetration - 15)
+
+	add_fingerprint(user)
+	if((HAS_TRAIT(user, TRAIT_CLUMSY)) && prob(50))
+		to_chat(user, "<span class ='danger'>You hit yourself over the head.</span>")
+		user.adjustStaminaLoss(stamina_force)
+
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			H.apply_damage(2*force, BRUTE, BODY_ZONE_HEAD)
+		else
+			user.take_bodypart_damage(2*force)
+		return
+	if(!isliving(target))
+		return ..()
+	if(iscyborg(target))
+		if (user.a_intent != INTENT_HARM)
+			playsound(get_turf(src), hitsound, 75, 1, -1)
+			user.do_attack_animation(target) // The attacker cuddles the Cyborg, awww. No damage here.
+			return
+	if (user.a_intent != INTENT_HARM)
+		force = non_harm_force
+	else
+		force = initial(force)
```

Comment:
This will tell different force value on examining everytime it's used with different intent.

Reply:
Is that a problem? It's going to deal different force depending on your intent

Reply:
Actually I do see the issue, force should be recalculated when the force string is returned. Should we refactor force into a more extensible system?

Reply:
honestly, yes, we should if we can.
`initial(force)` isn't a good practice for most items.
For example, you can trim blades to increase its force damage. Increased force damage will be removed if `initial(force)` is used, or directly takes an adjusted value.

![image](https://github.com/user-attachments/assets/bd4b6fac-e1d3-4240-b430-4621861c48f1)

(Chainsaw code)
As you can see, damage isn't reliable.